server.py

```diff
@@ -14,6 +14,8 @@ class Positions(Enum):
   PRESIDENT = "PRESIDENT"
   VICE_PRESIDENT = "VICE_PRESIDENT"
   TREASURER = "TREASURER"
+  SECRETARY = "SECRETARY"
+  TECHNICAL_OFFICER='TECHNICAL_OFFICER'
```

Comment:
Nitpick, but pycodestyle would complain about consistent quotation mark use and spacing here.

Reply:
Changes Made